src/opera_disp_tms/utils.py

```diff
@@ -103,10 +103,23 @@ def transform_point(x: float, y: float, source_wkt: str, target_wkt: str) -> Tup
 
 
 def check_bbox_all_int(bbox: Iterable[int]) -> None:
```

Comment:
`validate_bbox` would be a more descriptive name after these changes

Reply:
Done!